slang.h

```diff
@@ -1659,6 +1660,36 @@ extern "C"
         int                     targetIndex,
         ISlangSharedLibrary**   outSharedLibrary);
 
+    /** Get the output code associated with a specific target.
+
+    @param request   The request 
+    @param targetIndex The index of the target to get code for (default: zero).
+    @param outBlob A pointer that will receive the blob of code
+    @returns A `SlangResult` to indicate success or failure.
+
+    The lifetime of the output pointer is the same as `request`.
```

Comment:
This comment seems inaccurate; I assume the output blob is properly reference-counted and will be retained as long as the caller retains it.